.github/workflows/main.yml

```diff
@@ -99,6 +102,7 @@ jobs:
       with:
         name: test-results
         path: |
+          target/dependency-check-report.html
```

Comment:
This should IMO end in a separate artifact.
Background: Test-results are only uploaded, when all previous steps have succeeded but _not_ when e.g. tests failed. This happens, because steps have an implicit `if: success()`, which should either be an `if: always()` or probably better `if: failed()`.
OWASP-Reports however are only "interesting", when the build succeeds.

There's #1345 to fix the test-results artifact.

Reply:
good point, fixed.